test/worlds/CMakeLists.txt

```diff
@@ -6,3 +6,4 @@ configure_file (environmental_data.sdf.in ${PROJECT_BINARY_DIR}/test/worlds/envi
 
 configure_file (environmental_sensor.sdf.in ${PROJECT_BINARY_DIR}/test/worlds/environmental_sensor.sdf)
 configure_file (hydrodynamics.sdf.in ${PROJECT_BINARY_DIR}/test/worlds/hydrodynamics.sdf)
+configure_file (lighter_than_air_dynamics.sdf.in ${PROJECT_BINARY_DIR}/test/worlds/lighter_than_air_dynamics.sdf)
```

Comment:
Do we need to configure this? The `hydrodynamics.sdf.in` and `environmental_sensor.sdf.in` use cmake macros. All other test worlds just directly look for the file in the project source.

Reply:
WIP, and will add an example :+1:

Reply:
resolved